### DISCUSSION TITLE
How to avoid Fooocus download the model "juggernautXL_v8Rundiffusion.safetensors"

### DISCUSSION THREAD

**Discussion by suonnon** [lllyasviel/Fooocus#2681] [Q&A] [answered]
Hi, I use a folder that includes many checkpoint models for both A1111, ComfyUI,... And now I want to use it with Fooocus. I changed the path in config.txt. But Fooocus keep downloading the model juggernautXL_v8Rundiffusion.safetensors from huggingface. Is there a way to disable this?

**Comment by Rogala** [comment, marked as answer]
~\Fooocus\presets\*.json.}
Remove the part of the code that is responsible for loading checpoint when selecting a preset.

juggernautXL_v8Rundiffusion.safetensors is loaded when **default.json** and **lightning.json** is selected.

**Reply by suonnon** [reply]
@Rogala  Thanks. So I curious a bit, I set the default model to another model on config.txt , so it doesn't affect the default model  "juggernautXL_v8Rundiffusion.safetensors"? Change the default model on default.json is a must if I want to change the default model? Changing be settings on the GUI also doen't work?

**Reply by Rogala** [reply]
Yes, you can change the checkpoint to your preference in **default.json** or leave it empty.
"default_model": "",

**Reply by suonnon** [reply]
@Rogala  Thank you so much!
